csharp/generate/default_templates/modelGeneric.mustache

```diff
@@ -0,0 +1,351 @@
+    /// <summary>
```

Comment:
Why is this file called `modelGeneric` but the api mustache template file also in the `default_templates` folder is just called `api`. Is this difference in naming significant?

Reply:
This is just what OpenApi call it.